spec/comment_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Comments, type: :model do
```

Comment:
- [x] We now need to refactor the RSpec files too. Please rename `Comments` to `Comment`.

```suggestion
RSpec.describe Comment, type: :model do
```